### DISCUSSION TITLE
PATCH requests from the Microsoft Edge Browser do not seem to work...

### DISCUSSION THREAD

**Discussion by jthompson22** [supabase/supabase#17403] [Questions]
Hi all - we have an issue that I cannot seem to resolve. For reference - app is  React and Next with the latest version of Next Auth Helpers. Next is using Page Router, we've integrated the Supabase Client from NextAuthHelpers into the React auth helpers per the documentation. 

The above may or may not be related. 

All browsers seem to work fine except Microsoft Edge which, when we call a Supabase UPDATE method on our database using the Supabase JS client, throws an error saying that the PATCH method is not allowed. 

Specifically - 
"Access to fetch at .... from origin '....has been blocked by CORS policy: Method PATCH is not allowed by Access-Control-Allow-Methods in preflight response."

In the network tab, the preflight request for OPTIONS is coming in after the fetch request. Both Chrome and Firefox it comes before. 

Has anyone experienced this issue before and knows a workaround or fix? Seems like all other DB calls are working.

**Comment by GaryAustin1** [comment]
Check your browser extensions.  One at least blocks Patch by default...
https://github.com/orgs/supabase/discussions/17296-
